### DISCUSSION TITLE
可以通过微信图片dat的文件名反推原名吗？

### DISCUSSION THREAD

**Discussion by Rackar** [ttttupup/wxhelper#409] [General]
形如聊天记录中的3012169c0c2ce323d2a02a7085008eff.dat图片，另存是文件名为微信图片_20240504213346.jpg
不知道是用一个md5或者什么加密算法生成的，还是就是随机数然后数据库中绑定？
想做一个dat解密图片浏览器，想着能获取原名最好。

**Comment by ghost** [comment]
意义不太大，可以查询一下数据库看看，我觉得应该有原始文件名。

**Comment by ttttupup** [comment]
发送文件有原文件名，图片没有。
